radicle-node/src/service/message.rs

```diff
@@ -365,10 +376,22 @@ impl fmt::Debug for Message {
             Self::Announcement(Announcement { node, message, .. }) => {
                 write!(f, "Announcement({}, {:?})", node, message)
             }
+            Self::Ping { ponglen, zeroes } => write!(f, "Ping({ponglen}, {zeroes})"),
+            Self::Pong { zeroes } => write!(f, "Pong({zeroes})"),
         }
     }
 }
 
+// Should ZeroBytes be in wire/messsages instead?
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ZeroBytes(pub u16);
+
+impl fmt::Display for ZeroBytes {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "ZeroBytes({})", self.0)
+    }
+}
```

Comment:
Do we need a `Display` instance for this?

Reply:
It's used by Message's fmt::Debug, is there an alternative option?

Reply:
I would think implementing `fmt::Debug` would be enough?